content/en/blog/2024/go-opentelemetry-io.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Planned Migration for go.opentelemetry.io
+linkTitle: Planned migration for go.opentelemetry.io
+date: 2024-08-22
+author: >-
+  [Mike Dame](https://github.com/damemi) (Google),
+draft: true # TODO: remove this line once your post is ready to be published
+# canonical_url: http://somewhere.else/ # TODO: if this blog post has been posted somewhere else already, uncomment & provide the canonical URL here.
+body_class: otel-with-contributions-from # TODO: remove this line if there are no secondary contributing authors
+issue: 5086
+sig: Go SIG
+---
+
+With contributions from Tyler Yahn and Austin Parker.
+
+## TL;DR
+
+The app that serves requests to [go.opentelemetry.io] will be migrating to a new
+host on September 9, 2024. There are no changes required from you. We are making
```

Comment:
Should this include an approximate time? A date is very vague, especially with timezone differences.

Reply:
Added